core/src/test/java/io/questdb/test/griffin/AsOfJoinFuzzTest.java

```diff
@@ -70,7 +70,9 @@ private void assertResultSetsMatch0(Rnd rnd) throws Exception {
                 LimitType.values(),
```

Comment:
A question on the above `JoinType.values()` line. Why don't we fuzz keyed LT JOIN? There are two implementations: `LtJoinLightRecordCursorFactory` and `LtJoinRecordCursorFactory` which can be force via the `fullFatJoins` flag in `SqlCompiler`.

Reply:
I guess that was forgotten, I'll add it. thanks!